forem/forem
# Remove redundant sidekiq configuration

## What type of PR is this? (check all applicable)


## Description
I noticed duplicate INFO logs from sidekiq while running `bin/startup`. It looked like sidekiq was loading cron jobs twice. After some sleuthing, I realized this was because the Sidekiq configuration was manually loading the scheduler file. This is unnecessary because the sidekiq-cron gem automatically loads jobs from`config/scheduler.yml`, see: https://github.com/sidekiq-cron/sidekiq-cron?tab=readme-ov-file#adding-cron-job.

More than just removing duplicates, I was motivated to make this change because reducing lines makes the log output for `bin/startup` easier to parse 😸 

## QA Instructions, Screenshots, Recordings

Run `bin/startup` before this change, notice the two big chunks of `INFO: Cron Jobs - added job with name: <job-name>` logs. 
Run `bin/startup` after this change, notice only one big chunk of those logs. 

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [x] No, and this is why: no current test coverage for sidekiq initialization 


![choppy](https://github.com/forem/forem/assets/13890477/8ca2398e-f3c9-4aab-871a-9321288d6916)
